src/components/TestimonialSlider.jsx

```diff
@@ -10,7 +10,8 @@ import Testimonial from "./Testimonial";
 const TestimonialSlider = () => {
   const [slider, setSlider] = useState(1);
   const [timeoutActive, setTimeoutActive] = useState(true);
-  const { testimonials } = useSelector((testimonials) => testimonials.appData);
+  const { testimonials } = useSelector((state) => state.appData);
+  //const { testimonials } = useSelector((testimonials) => testimonials.appData);
```

Comment:
maybe remove the comment or the other one and comment in the comment?
Since it's ```(contact) => contact.appData``` in the contact file maybe the commented out version is better to comment in and remove the other?

Reply:
damn.. I'll remove the comment. I actually don't get how the commented version works))

Reply:
fixed

Reply:
haha i thought it was you that asked me to change in my pr to contact instead of state x) or maybe it was me who misunderstood you